src/compare_tracker_with_nxn_sheet.py

```diff
@@ -17,53 +17,207 @@
+
+
+    # Retrieve DOIs and adjust them
+    valentine_dois = set([data[v_doi_index] for data in valentines_sheet])
+    tracking_sheet_dois = set([track[t_doi_index] for track in tracking_sheet if track[t_doi_index]])
     tracking_sheet_dois = {f"10.{doi.split('doi.org/10.')[-1]}" for doi in tracking_sheet_dois if "doi.org" in doi}
 
     unregistered_dois = valentine_dois - tracking_sheet_dois
 
-    unregistered_table = [row for row in valentines_sheet if row[1] in unregistered_dois]
+    # Translate unregistered dois into the table
+    unregistered_table = [row for row in valentines_sheet if row[v_doi_index] in unregistered_dois]
 
-    valentine_accessions = set([data[11] for data in unregistered_table if data[11]])
-    tracking_sheet_accessions = set([track[1] for track in tracking_sheet if track[1]])
+    # Retrieve accessions and repeat filtering
+    valentine_accessions = set([data[v_data_location_index] for data in unregistered_table if data[v_data_location_index]])
+    tracking_sheet_accessions = set([track[t_data_location_index] for track in tracking_sheet if track[t_data_location_index]])
 
     unregistered_accessions = valentine_accessions - tracking_sheet_accessions
 
-    second_unregistered_table = [row for row in unregistered_table if row[11] in unregistered_accessions or not row[11]]
+    second_unregistered_table = [row for row in unregistered_table if row[v_data_location_index] in unregistered_accessions or not row[v_data_location_index]]
 
-    valentine_titles = set([data[4].lower() for data in second_unregistered_table if data[4]])
-    tracking_sheet_titles = set([track[14].lower() for track in tracking_sheet if track[14]])
+    valentine_titles = set([data[v_title_index].lower() for data in second_unregistered_table if data[v_title_index]])
+    tracking_sheet_titles = set([track[t_title_index].lower() for track in tracking_sheet if track[t_title_index]])
 
     unregistered_titles = valentine_titles - tracking_sheet_titles
 
-    full_unregistered_table = [row for row in second_unregistered_table if row[4].lower() in unregistered_titles]
+    full_unregistered_table = [row for row in second_unregistered_table if row[v_title_index].lower() in unregistered_titles]
 
     return full_unregistered_table
 
 
-def filter_table(valentines_table):
-    filtered_table = [row for row in valentines_table if row[8].lower() in ['human', 'human, mouse', 'mouse, human']]
+def filter_table(valentines_table, full_database):
+    """
+    Filter the table based on organism/technology criteria
+    :param valentines_table:
+    :return:
+    """
+    organism_index = find_header_index(full_database, 'Organism')
+    technique_index = find_header_index(full_database, 'Technique')
+    measurement_index = find_header_index(full_database, 'Measurement')
+
+    filtered_table = [row for row in valentines_table if row[organism_index].lower() in ['human', 'human, mouse', 'mouse, human']]
```

Comment:
the options we have in the tracker are: human, mouse, human&mouse, other, human&other, can we make it automated to change `human, mouse` and `mouse, human` to `human&mouse`?
Along similar lines, would it be possible to drop spaces between the organs, i.e. replace `, ` with `,`

Reply:
Yeah, this is just to filter rows but I can add it on the "print_output" function